tls/s2n_record.h

```diff
@@ -81,7 +81,7 @@
 S2N_RESULT s2n_record_max_write_size(struct s2n_connection *conn, uint16_t max_fragment_size, uint16_t *max_record_size);
 extern S2N_RESULT s2n_record_max_write_payload_size(struct s2n_connection *conn, uint16_t *max_fragment_size);
 extern S2N_RESULT s2n_record_min_write_payload_size(struct s2n_connection *conn, uint16_t *payload_size);
-extern int s2n_record_write(struct s2n_connection *conn, uint8_t content_type, struct s2n_blob *in);
+S2N_RESULT s2n_record_write(struct s2n_connection *conn, uint8_t content_type, struct s2n_blob *in);
```

Comment:
This macro is not `extern` but the declaration it replaces is. Should this be `extern S2N_RESULT`?

Reply:
It's the method that was being labeled extern, not a macro, and it's completely unnecessary to label methods extern. The extern is implied.